HW_4/task_3.py

```diff
@@ -0,0 +1,62 @@
+"""
+Задать натуральное число k.
+Сформируйте многочлен (полином) степени k со случайными коэффициентами из промежутка от 0 до 100, включительно.
+Многочлен вывести в консоль и записать в файл.
+
+Ввод: значение типа <int>
+Вывод: значение типа <str>, файл с одной строкой.
+
+Пример:
+2
+2x^2 + 4x + 5 = 0
+"""
+import random
+print('Введите степень k для многочлена ax^k + bx + c = 0')
+k = input()
```

Comment:
Это сообщение можно было сразу написать внутри функции input?
```suggestion
k = input('Введите степень k для многочлена ax^k + bx + c = 0')
```